frontend/src/components/NavBar.js

```diff
@@ -359,7 +359,12 @@ class NavBar extends React.Component<{
                                 <Link to='/account/signup'>
                                     <Button color='inherit'>Sign Up</Button>
                                 </Link>
-                                <Link to='/account/login'>
+                                <Link
```

Comment:
Should we do the same for the sign up button?

Reply:
I think it's outside the scope of this issue. But we can talk about it in the meeting? Thanks!